src/test/data/JsonSerializableAddressBookTest/invalidPersonAddressBook.json

```diff
@@ -1,6 +1,5 @@
 {
   "persons": [ {
-    "name": "Hans Muster",
-    "phone": "948dfa2424"
+    "name": "Ha!ns Mu@ster"
```

Comment:
I understand that this is to cater to some Tests that are expected to fail due to invalidPerson right?

Reply:
That is correct.